src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -191,6 +192,20 @@ function describeID(id: DebugID): string {
   return describeComponentFrame(name, element && element._source, ownerName);
 }
 
+function describeFiber(fiber : Fiber) : string {
+  var owner = fiber._debugOwner;
+  var source = fiber._debugSource;
+  if (owner == null && source == null) {
+    return '';
```

Comment:
Even if we don't have owner and source, we still want the name of this fiber, right?